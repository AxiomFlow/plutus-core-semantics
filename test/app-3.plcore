(module Foo (imported) (exported () ()) (local () ())
  (define r2
    [(lam x (lam y (builtin addInt x y))) 2 3 ])
)
