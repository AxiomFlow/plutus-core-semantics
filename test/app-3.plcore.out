<T> <k> . </k> <env> ( Foo . r2 ) |-> 5 </env> <mod> Foo </mod> </T>
